src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -60,7 +60,9 @@
  (fn [{:keys [db]} [{:keys [address token recipient stack-id]}]]
    (let [[prefix to-address] (utils/split-prefix-and-address address)
          prefix-seq          (string/split prefix #":")
-         selected-networks   (mapv #(utils/short-name->id (keyword %)) prefix-seq)]
+         selected-networks   (->> prefix-seq
+                                  (remove string/blank?)
```

Comment:
It'd be good to have this logic as a separate function that takes an address and returns the networks found in it. I think it'll be a common functionality.

We can do that in a later refactor

Reply:
Might be good to track this in an issue 👌